src/orchestrator/core/flow/Projects.py

```diff
@@ -284,6 +284,102 @@ def update_project(self,
 
         return PROJECT, DOMAIN_NAME, PROJECT_NAME
 
+
+    def rename_project(self,
+                       DOMAIN_ID,
+                       DOMAIN_NAME,
+                       PROJECT_ID,
+                       PROJECT_NAME,
+                       ADMIN_USER,
+                       ADMIN_PASSWORD,
+                       ADMIN_TOKEN,
+                       NEW_SUBSERVICE_NAME):
+
+        '''Rename Project Name
+
+        In case of HTTP error, return HTTP error
+
+        Params:
+        - DOMAIN_ID: id of domain
+        - DOMAIN_NAME: name of domain
+        - PROJECT_ID: id of project
+        - PROJECT_NAME: name of project
+        - SERVICE_ADMIN_USER: Service admin username
+        - SERVICE_ADMIN_PASSWORD: Service admin password
+        - SERVICE_ADMIN_TOKEN: Service admin token
+        - NEW_SUBSERVICE_NAME: New subservice name
+        Return:
+        - project detail
+        '''
+        data_log = {
+            "DOMAIN_ID": "%s" % DOMAIN_ID,
+            "DOMAIN_NAME": "%s" % DOMAIN_NAME,
+            "PROJECT_ID": "%s" % PROJECT_ID,
+            "PROJECT_NAME": "%s" % PROJECT_NAME,
+            "ADMIN_USER": "%s" % ADMIN_USER,
+            "ADMIN_PASSWORD": "%s" % "***", #ADMIN_PASSWORD,
+            "ADMIN_TOKEN": self.get_extended_token(ADMIN_TOKEN),
+            "NEW_SUBSERVICE_NAME": "%s" % NEW_SUBSERVICE_NAME,
+        }
+        self.logger.debug("FLOW rename_project invoked with: %s" % json.dumps(
+            data_log,
+            indent=3)
+        )
+        try:
+            if not ADMIN_TOKEN:
+                if not DOMAIN_ID:
+                    ADMIN_TOKEN = self.idm.getToken(DOMAIN_NAME,
+                                                    ADMIN_USER,
+                                                    ADMIN_PASSWORD)
+                    DOMAIN_ID = self.idm.getDomainId(ADMIN_TOKEN,
+                                                     DOMAIN_NAME)
+                else:
+                    ADMIN_TOKEN = self.idm.getToken2(DOMAIN_ID,
```

Comment:
Maybe getToken2 name could be improved?

Reply:
done in PR https://github.com/telefonicaid/orchestrator/pull/333

Reply:
Merged PR #333. I understand this PR needs to be synched with master.